c/list-simple/README.txt

```diff
@@ -0,0 +1,34 @@
+Contributed by Johannes Knaut
+
+This set contains C programs that apply basic manipulations to a simple linked list.
+
+Common in all programs:
+- size = 2 for the initial list
+- data = 1 for each initial list node
+- if manipulation result is unexpected, an error function is called
+
+Varying:
+- number of links: sll / dll
+- circularity: circular(c) / NULL-terminated(n)
+- applied manipulation:
+append-equal(appeq) / append-unequal(appuneq) / prepend-equal(prepeq) / prepend-unequal(prepuneq) / insert-mid-equal(inseq) /
+insert-mid-unequal(insuneq) / remove-all(rmall) / remove-all-reverse(rmallr) / update-all(updall) / update-all-reverse(updallr)
+
+Manipulations with expected results:
+- appeq(1-1, 1) = 1-1-1'
+- appuneq(1-1, 5) = 1-1-5
+- inseq(1-1, 1) = 1-1'-1
+- insuneq(1-1, 5) = 1-5-1
+- prepeq(1-1, 1) = 1'-1-1
+- prepuneq(1-1, 5) = 5-1-1
+- rmall(1-1) -> '-1 -> NULL
+- rmallr(1-1) -> 1-' -> NULL
```

Comment:
Could you explain what the `'` character means in this notation?

Reply:
It was meant to be a visual clue to see the different order in which nodes are removed.

Reply:
And to distinguish the inserted 1s from the others.

Reply:
It would be nice to also find this information in the apart from that very well written README :wink:.

Reply:
Thanks, i will add some info.